checker/build.xml

```diff
@@ -383,6 +379,7 @@
                 <include name="**/UnitsTools.java"/>
                 <include name="**/SignednessUtil.java"/>
                 <include name="**/I18nFormatUtil.java"/>
+                <include name="${checker.jdk8orhigher.sources}" if="${jdk8orhigher}"/>
```

Comment:
I think these source files should be copied to checker-qual-source.jar regardless of the JDK version, and the file should be copied to checker-qual.jar if it can be made, which is if JDK 8 is available regardless of what the default version of the JDK is.